### PR TITLE
Add dataSetId support to Template View

### PR DESCRIPTION
**Pull request by Tapped**
Add support for adding a dataSetId to a View

### CODE REVIEW

File: cognite/client/data_classes/templates.py [hunk 1]
```diff
@@ -312,18 +312,21 @@ class View(CogniteResource):
     Args:
         external_id (str): The external ID provided by the client. Must be unique for the resource type.
         source (Source): Defines the data source for the view.
+        data_set_id (int): The dataSetId of the view
```
Comment (Tapped) [inline cognite/client/data_classes/templates.py:315]: I guess this should be optional in the type hint interface, right?

**Comment by Tapped** [comment]
@erlendvollset Would you have time to look at this one later today?

**Review by erlendvollset — APPROVED** [review]
Sorry, I don't get notifications on GitHub comments. LGTM.

**Comment by erlendvollset** [comment]
You may want to bump version also, to have it released?

**Comment by Tapped** [comment]
@erlendvollset Do you know why I'm seeing this:
```
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.5.10/x64/bin/pipenv", line 5, in <module>
    from pipenv import cli
  File "/opt/hostedtoolcache/Python/3.5.10/x64/lib/python3.5/site-packages/pipenv/__init__.py", line 55, in <module>
    from .cli import cli
  File "/opt/hostedtoolcache/Python/3.5.10/x64/lib/python3.5/site-packages/pipenv/cli/__init__.py", line 1, in <module>
    from .command import cli    # noqa
  File "/opt/hostedtoolcache/Python/3.5.10/x64/lib/python3.5/site-packages/pipenv/cli/command.py", line 107
    echo(f"  - {crayons.normal(key, bold=True)}")
                                               ^
SyntaxError: invalid syntax
``` in the 3.5 build

**Pull request merged**